src/Sylius/Bundle/CoreBundle/Resources/config/services/shop_fixtures.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+use Sylius\Bundle\CoreBundle\ShopFixtures\Symfony\Messenger\CommandBusInterface;
+use Sylius\Bundle\CoreBundle\ShopFixtures\Symfony\Messenger\MessengerCommandBus;
+
+return static function (ContainerConfigurator $container) {
+    $container->import('shop_fixtures/command_handlers.php');
+    $container->import('shop_fixtures/default_values.php');
+    $container->import('shop_fixtures/factories.php');
+    $container->import('shop_fixtures/fixtures.php');
+    $container->import('shop_fixtures/stories.php');
+
+    $container->services()
+
+        ->set('sylius.shop_fixtures.bus.command', MessengerCommandBus::class)
+            ->args([
+                service('sylius.shop_fixtures.command_bus')
+            ])
+        ->alias(CommandBusInterface::class, 'sylius.shop_fixtures.bus.command')
+
+    ;
```

Comment:
```suggestion
    $container->services()
        ->set('sylius.shop_fixtures.bus.command', MessengerCommandBus::class)
            ->args([
                service('sylius.shop_fixtures.command_bus')
            ])
        ->alias(CommandBusInterface::class, 'sylius.shop_fixtures.bus.command')
    ;
```

or

```suggestion
    $services = $container->services();

    $services->set('sylius.shop_fixtures.bus.command', MessengerCommandBus::class)
        ->args([
            service('sylius.shop_fixtures.command_bus')
        ])
        ->alias(CommandBusInterface::class, 'sylius.shop_fixtures.bus.command')
    ;
```

It looks weird for me currently :/. This suggestion applies to all PHP configs :P.

Reply:
https://github.com/symfony/symfony/blob/6.3/src/Symfony/Bundle/FrameworkBundle/Resources/config/notifier.php